### PR TITLE
Platform

### PR DESCRIPTION
**Pull request by GSuma21**

### CODE REVIEW

File: src/app/shared/components/join-dialog-box/join-dialog-box.component.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                <p fxLayoutAlign="start center" >Meeting id: </p>
+            </div>
+            <div>
+                <p fxLayoutAlign="start center" class="heading">{{data.result?.meta?.meetingId}} <mat-icon fxLayoutAlign="start center" class="mat-icon-content-copy" (click)="copyToClipBoard()">content_copy</mat-icon></p>
+            </div>
+        </div>
+       
+    </div>
+    <div>
+        <button mat-raised-button class="join-button" (click)="onClickJoin()">join</button>
```
Comment (kiranharidas187) [inline src/app/shared/components/join-dialog-box/join-dialog-box.component.html]: translate

File: src/app/shared/components/join-dialog-box/join-dialog-box.component.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                <p fxLayoutAlign="start center" >Password: </p>
+            </div>
+            <div>
+                <p fxLayoutAlign="start center" class="heading">{{data.result?.meta?.password}} <mat-icon fxLayoutAlign="start center" class="mat-icon-content-copy" (click)="copyToClipBoard()">content_copy</mat-icon></p>
+            </div>
+        </div>
+        <div fxLayout="row" fxLayoutGap="10px">
+            <mat-icon fxLayoutAlign="start center">vpn_key</mat-icon>
+            <div>
+                <p fxLayoutAlign="start center" >Meeting id: </p>
```
Comment (kiranharidas187) [inline src/app/shared/components/join-dialog-box/join-dialog-box.component.html]: translate

File: src/deployment/ansible.yml [hunk 3]
```diff
@@ -29,7 +29,7 @@
     - name: Change directory
       shell: chdir {{project_path}}
     - name: Delete old pm2 process
-      shell: pm2 delete MentorEd-portal
+      shell: pm2 delete MentorEd-Portal
```
Comment (kiranharidas187) [inline src/deployment/ansible.yml]: is these changes required

Comment (Cafnanc) [reply src/deployment/ansible.yml:32]: yes kiran. this was suggested by rakesh
